### PR TITLE
Update localization.md

### LINKED ISSUE #8244: Generic Class and Localization blog
See [Localization & Generics](http://www.hishambinateya.com/localization-and-generics)

and https://github.com/aspnet/Home/issues/3422

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d1833676-ff1b-c587-cc9c-81afffa8ef68
* Version Independent ID: ad989395-3f49-4d55-521e-cf6fe56281ee
* Content: [Globalization and localization in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/localization?view=aspnetcore-2.1)
* Content Source: [aspnetcore/fundamentals/localization.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/fundamentals/localization.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #8244

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@hishamco  please review.
Do you have a link to the blog I can add?

**Comment by hishamco** [comment]
I already referred to that in the original discussion http://www.hishambinateya.com/localization-and-generics, but the link is broken now because I have an issue with the hosting, but it's available [here](https://github.com/hishamco/hishambinateya.com/blob/master/Posts/localization-and-generics.md) too

File: aspnetcore/fundamentals/localization.md [hunk 1]
```diff
@@ -343,4 +343,5 @@ Terms:
 * [Globalizing and localizing .NET applications](/dotnet/standard/globalization-localization/index)
 * [Resources in .resx Files](/dotnet/framework/resources/working-with-resx-files-programmatically)
 * [Microsoft Multilingual App Toolkit](https://marketplace.visualstudio.com/items?itemName=MultilingualAppToolkit.MultilingualAppToolkit-18308)
+* [Localization & Generics](https://github.com/hishamco/hishambinateya.com/blob/master/Posts/localization-and-generics.md)
 * [Generic Class and Localization](https://github.com/aspnet/AspNetCore.Docs/issues/8244)
```
Comment (hishamco) [inline aspnetcore/fundamentals/localization.md]: I'm not sure if we need this one,

Comment (Rick-Anderson) [reply aspnetcore/fundamentals/localization.md]: Delete https://github.com/aspnet/AspNetCore.Docs/issues/8244 ?

Comment (hishamco) [reply aspnetcore/fundamentals/localization.md]: Yep, because the article is answer for that issue

Comment (Rick-Anderson) [reply aspnetcore/fundamentals/localization.md]: I thought so but the blog is highlighted in GitHub.

**Pull request merged**